src/yang/ietf-bgp-policy.yang

```diff
@@ -191,6 +191,33 @@ module ietf-bgp-policy {
     container bgp-defined-sets {
       description
         "BGP-related set definitions for policy match conditions.";
+      container as-path-sets {
+        description
+          "Enclosing container for list of define AS path sets.";
+        list as-path-set {
+          key "name";
+          description
+            "List of defined AS path sets.";
+          leaf name {
+            type string;
+            description
+              "Name of the AS path set -- this is used to reference
+               the set in match conditions.";
+          }
+          leaf-list member {
+            type string;
+            description
+              "AS path regular expression using BGP YANG AS_PATH
```

Comment:
The description says 'member' is a string with a AS path regular expression. But there is no pattern statement here to reflect the AS path regular expression. I see that we have a reference to the new section on AS PATH regular expression, but that is informative. Should this explain (or have an issue) to track if/when we do define a regular expression.

Reply:
As we had discussed, it makes zero sense to try to list the rules for how to create a regular expression in general, much less the subset we're creating for such a language element, in the leaf.  So, we're not going to have anything here.

I'm rusty on my compiler theory, but it's my memory that you can't express the output of fully arbitrary balanced parenthesis matches as a regular expression.  I think you need a CFG for that.  Thus, I don't think we can use a YANG leaf pattern to help here.